src/directory/directory.mjs

```diff
@@ -465,6 +465,17 @@ export const directory = {
                 }
               ]
             },
+            {
+              path: 'src/pages/[platform]/build-a-backend/existing-resources/index.mdx',
+              children: [
+                {
+                  path: 'src/pages/[platform]/build-a-backend/existing-resources/cli/index.mdx'
+                }
+                // {
```

Comment:
is this intended?  It looks like the page is included in this PR but with this commented out it won't be in the menu

Reply:
There's a second page that focuses on how to add existing resources using the CDK, where the one we're publishing in this PR is focused on doing that with the CLI.